docs/docs/delegators/delegator-guide-cli.md

```diff
@@ -7,45 +7,32 @@ This document contains all the necessary information for delegators to interact
 
 It also contains instructions on how to manage accounts, restore accounts from the fundraiser and use a ledger nano device.
 
-:::warning
-**Very Important**: Please assure that you follow the steps described hereinafter
```

Comment:
Should we keep some type of a disclaimer?

Reply:
@MSalopek asked me to remove it.

Reply:
Not to remove it, bump it down to a `:::tip` or `:::info` so it's not red